SPECS/qttools/qttools.spec

```diff
@@ -53,60 +53,60 @@ Requires: %{name}-devel%{?_isa} = %{version}-%{release}
 %{summary}.
 
 %package libs-designer
-Summary: Qt5 Designer runtime library
+Summary: Qt6 Designer runtime library
 Requires: %{name}-common = %{version}-%{release}
 %description libs-designer
 %{summary}.
 
 %package libs-designercomponents
-Summary: Qt5 Designer Components runtime library
+Summary: Qt6 Designer Components runtime library
 Requires: %{name}-common = %{version}-%{release}
 %description libs-designercomponents
 %{summary}.
 
 %package libs-help
-Summary: Qt5 Help runtime library
+Summary: Qt6 Help runtime library
 Requires: %{name}-common = %{version}-%{release}
 # when split happened
-Conflicts: qt5-tools < 5.4.0-0.2
+Conflicts: qttools < 5.4.0-0.2
```

Comment:
I'm thinking we probably don't need this line?

Reply:
removed!